internal_transfers/actions/tests/database.spec.ts

```diff
@@ -237,13 +241,56 @@ describe("All Database Tests", () => {
         },
       ]);
     });
+    test("markReceiptProcessed does nothing when hash doesn't exist", async () => {
+      await expect(
+        markReceiptProcessed(db, "0x01")
+      ).resolves.not.toThrowError();
+    });
```

Comment:
Out of curiosity: Should we ever be expecting to mark a receipt as processed when there isn't a hash for it?

Reply:
Yes, in the scenario where we run the program manually to "backfill" the data from many months ago till now, we will be calling this program on records that were never part of the "tx_receipts" table.